pipeline/metadata/4.3.yaml

```diff
@@ -0,0 +1,534 @@
+# =====================================================================================
+# Metadata file for 4.3 RHCS release version execution for pipeline V3.
+# Single file to execute the all tier-1, tier-2, tier-3 and tier-4 suites in parallel
+# as per defined in each stages.
+# Each stage will execute in sequential pattern.
+# Contains all Default parameter used for Execution when no override is specified.
+# parameter requied to create a similar custom file is suite name, suite yaml file, global configuration file,
+# platform, rhbuild, inventory and metadata information like frequency of execution, tier, cloud type, functional group and stage.
+# =====================================================================================
+- name: "test-cephfs-extended-mat"
+  suite: "suites/nautilus/cephfs/tier-1_fs.yaml"
+  global-conf: "conf/nautilus/cephfs/tier-1_fs.yaml"
+  platform: "rhel-8"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - cephfs
+    - stage-1
+- name: "test-rbd-extended-mat"
+  suite: "suites/nautilus/rbd/tier-1_rbd.yaml"
+  global-conf: "conf/nautilus/rbd/tier-0_rbd.yaml"
+  platform: "rhel-8"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - rbd
+    - stage-1
+- name: "test-rgw-ssl-rpm"
+  suite: "suites/nautilus/rgw/tier-1_rgw_ssl_regression.yaml"
+  global-conf: "conf/nautilus/rgw/tier-1_rgw.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7.9-server-x86_64.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7.9-minimal-amd64-3.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - rgw
+    - stage-1
+- name: "test-upgrade-on-container-based-deployments"
+  suite: "suites/nautilus/upgrades/tier-1_upgrade_container.yaml"
+  global-conf: "conf/nautilus/upgrades/tier-1_upgrade.yaml"
+  platform: "rhel-8"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - upgrades
```

Comment:
Upgrades is not an FG in itself. So this test suite can contain dmfg also as another tag along with upgrades.

Reply:
I can add as upgrades/dmfg   ...right?

Reply:
Done added as  
- upgrades
- dmfg